### PR TITLE
[FIXED JENKINS-25637] Exclude the plugin root path from requiring crumb

### PR DESCRIPTION
**Pull request by s0undt3ch**
Refs https://issues.jenkins-ci.org/browse/JENKINS-25637

### CODE REVIEW

**Comment by s0undt3ch** [comment]
Sorry for not including tests but I'm not a java developer. This was added by looking how other plugins have handled this problem.

File: src/main/java/org/jenkinsci/plugins/build_token_root/BuildRootAction.java [hunk 1]
```diff
@@ -100,10 +104,10 @@ public void doBuildWithParameters(StaplerRequest req, StaplerResponse rsp, @Quer
...
         List<ParameterValue> values = new ArrayList<ParameterValue>();
         for (ParameterDefinition d : pp.getParameterDefinitions()) {
-        	ParameterValue value = d.createValue(req);
-        	if (value != null) {
-        		values.add(value);
-        	}
+            ParameterValue value = d.createValue(req);
+            if (value != null) {
+                values.add(value);
+            }
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/build_token_root/BuildRootAction.java]: Please avoid gratuitous whitespace changes in pull requests.

**Comment by jglick** [comment]
The patch is probably right but there is no functional test proving it.

**Comment by s0undt3ch** [comment]
Please see [my comment above](https://github.com/jenkinsci/build-token-root-plugin/pull/7#issuecomment-91644857)

File: src/main/java/org/jenkinsci/plugins/build_token_root/BuildRootAction.java (hunk 1 above)
Comment (s0undt3ch) [inline src/main/java/org/jenkinsci/plugins/build_token_root/BuildRootAction.java]: There were tabs in there while most of the file was spaces.

**Comment by ydubreuil** [comment]
@jglick I've added a test case to this patch in #10

**Comment by s0undt3ch** [comment]
Thanks!

**Pull request closed without merging**